<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuracao</class>
 <widget class="QMainWindow" name="Configuracao">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>750</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>460</width>
        <height>270</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1366</width>
        <height>1366</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabServidor">
       <attribute name="title">
        <string>Servidor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Servidor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineHost">
            <property name="minimumSize">
             <size>
              <width>253</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>22</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Porta:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Usuário:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineUser"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="5">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string> Senha: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="linePass">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="5">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Diretório de trabalho:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineDiretorio">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>103</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0" colspan="5">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="bntTestConnection">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>testar conexão</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>180</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bntSalvarConf">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Salvar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinPorta">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>65536</number>
          </property>
          <property name="value">
           <number>21</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSourceFile">
       <attribute name="title">
        <string>Arquivos e Pastas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="bntAdicionar">
            <property name="text">
             <string>Adicionar Arquivos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bntAdcionarDiretorio">
            <property name="text">
             <string>Adicionar Diretório</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bntRemover">
            <property name="text">
             <string>Remover Item</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bntLimpar">
            <property name="text">
             <string>Limpar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QProgressBar" name="progressFile">
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%v Arquivo Corrente</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QProgressBar" name="progressTotal">
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%m Progresso Total</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>285</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnEnviar">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Enviar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCancelar">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Cancelar</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTreeWidget" name="listFile">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>0</number>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>130</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Arquivos Remotos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="listFileRemoto">
          <property name="columnCount">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineHost</sender>
   <signal>editingFinished()</signal>
   <receiver>bntSalvarConf</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinPorta</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bntSalvarConf</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineUser</sender>
   <signal>editingFinished()</signal>
   <receiver>bntSalvarConf</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>linePass</sender>
   <signal>editingFinished()</signal>
   <receiver>bntSalvarConf</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineDiretorio</sender>
   <signal>editingFinished()</signal>
   <receiver>bntSalvarConf</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
